iterators_task_1/student/src/impl.cpp

```diff
@@ -1,2 +1,14 @@
 #include "impl.h"
 
+std::string reverseWords(const std::string& input) {
+	std::stringstream ss(input);
+	std::string reversed, word;
+	while (ss >> word) {
```

Comment:
that's acceptable way, but try to achieve this via the algorithms + iterators now